Neo4j.Driver/Neo4j.Driver/Public/IRecord.cs

```diff
@@ -28,11 +28,49 @@ public interface IRecord
     /// <summary>Gets the value specified by the given key.</summary>
     /// <param name="key">The key</param>
     /// <returns>the value specified with the given key.</returns>
-    object this[string key] { get; }
+    new object this[string key] { get; }
```

Comment:
we could drop this as it is defined on the IReadOnlyDictionary<string, object> interface and behaves the same.

Reply:
Done.